src/web/services/table-comparator.service.ts

```diff
@@ -37,38 +37,49 @@ export class TableComparatorService {
     return 0;
   }
 
+  /**
+   * Compares two strings which are expected to be floating point numbers depending on the order given.
```

Comment:
No need to specify it as "floating point" because all numbers behave like that in JS.